src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6182,4 +6182,8 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
     <comment>Do not localize project.json and `PackageReference` 
 0 - project path (project.json file path)</comment>
   </data>
+  <data name="PackageCommandDefaultSymbolPackageFormatChanging" xml:space="preserve">
```

Comment:
Is it possible to create only one string resource for this warning?

A possible place for this string would be NuGet.Packaging